config.js

```diff
@@ -91,8 +91,10 @@ config.perseoCore = {
  * NEXT EPL core options (with HA)
  */
 config.nextCore = {
+    /*
```

Comment:
I would add a comment in the following style:

"Note this parameters is empty, so Perseo will not use HA by default"